openslides/users/projector.py

```diff
@@ -12,8 +16,22 @@
 def user_slide(all_data: AllData, element: Dict[str, Any]) -> Dict[str, Any]:
     """
     User slide.
+
+    The returned dict can contain the following fields:
+    * user
     """
-    return {"error": "TODO"}
+    user_id = element.get("id")
+
+    if user_id is None:
+        return {"error": "id is required for user slide"}
```

Comment:
Please use the ProjectorElementException inserted with #4194 

See also line 31 in this file